bsp/x86/bootsector.s

```diff
@@ -3,7 +3,7 @@
 
 CYLS = 10
 BOOTSEG =   0x07C0
-INITSEG =   0x7000
+INITSEG =   0x0050
```

Comment:
use #include <common.h> and remove INITSEG here.
BTW: the file name bootsector.s to bootsector.S.